Algorithms/Searching/binary_search.py

```diff
@@ -0,0 +1,34 @@
+// Name : Nabanita Dash
```

Comment:
This has to be a comment in python. `//` in beginning would give compilation error. Fix it in all the files.

Reply:
My bad